NitroxClient/GameLogic/Entities.cs

```diff
@@ -50,18 +52,32 @@ public void Spawn(List<Entity> entities)
         {
             foreach (Entity entity in entities)
             {
+                GameObject batchRoot = EnsureBatchRoot(entity);
                 if (!alreadySpawnedIds.Contains(entity.Id))
                 {
-                    Spawn(entity, Optional<GameObject>.Empty());
+                    Spawn(entity, Optional<GameObject>.Empty(), batchRoot);
```

Comment:
If we were to remove alreadySpawnedIds would that fix the disappearing issue?

Reply:
unfortunately it needs changes on both server and client but it would(still an issue of the gameObjects getting assigned to the Waiters)